cmd/appeal.go

```diff
@@ -260,9 +270,10 @@ func rejectApprovalStepCommand(c *app.CLIConfig) *cobra.Command {
 		},
 	}
 
-	cmd.Flags().UintVar(&id, "id", 0, "appeal id")
+	cmd.Flags().UintVar(&id, "id", 0, "ID of the appeal")
 	cmd.MarkFlagRequired("id")
-	cmd.Flags().StringVarP(&approvalName, "approval-name", "a", "", "approval name going to be approved")
+
+	cmd.Flags().StringVarP(&approvalName, "approval-name", "a", "", "Approval name going to be approved")
```

Comment:
@rahmatrhd approval-name is the name of the approval step or the name of the person who is approving?

Reply:
name of the approval step

Reply:
Then lets call the flag step?  `approval-name` is bit confusing. WDYT?

Reply:
yeah, that sounds better 👍

Reply:
@rahmatrhd done. Please check now.